configs/TBS_LUCID_H7/config.h

```diff
@@ -177,7 +177,8 @@
 #define PINIO1_CONFIG 129
 #define PINIO2_BOX 41
 #define PINIO2_CONFIG 129
-
+#define PINIO3_BOX 42
+#define PINIO3_CONFIG 129
```

Comment:
Perhaps add a label for the pinio's, like

#define BOX_USER1_NAME       "CAM 1,2"
#define BOX_USER2_NAME       "PWR SW"

Reply:
very nice, thanks! is there a character limit on these?